core/input/input_map.h

```diff
@@ -66,6 +66,11 @@ class InputMap : public Object {
 	TypedArray<InputEvent> _action_get_events(const StringName &p_action);
 	TypedArray<StringName> _get_actions();
 
+#ifndef DISABLE_DEPRECATED
+	void _add_action_bind_compat_97281(const StringName &p_action, float p_deadzone);
```

Comment:
```suggestion
	void _add_action_bind_compat_97281(const StringName &p_action, float p_deadzone = 0.5);
```
Just to make it clearer

Reply:
These should also be under `protected` not `private`